cts-java/src/test/java/org/ga4gh/cts/api/variants/CallSetsPagingIT.java

```diff
@@ -0,0 +1,150 @@
+package org.ga4gh.cts.api.variants;
+
+import junitparams.JUnitParamsRunner;
+import org.apache.avro.AvroRemoteException;
+import org.ga4gh.ctk.transport.URLMAPPING;
+import org.ga4gh.ctk.transport.protocols.Client;
+import org.ga4gh.cts.api.Utils;
+import org.ga4gh.methods.GAException;
+import org.ga4gh.methods.SearchCallSetsRequest;
+import org.ga4gh.methods.SearchCallSetsResponse;
+import org.ga4gh.models.CallSet;
+import org.ga4gh.models.VariantSet;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests paging through {@link CallSet}s.
+ *
+ * @author Herb Jellinek
+ */
+@RunWith(JUnitParamsRunner.class)
+@Category(VariantsTests.class)
+public class CallSetsPagingIT {
+
+    private static final URLMAPPING urls = URLMAPPING.getInstance();
+
+    private static Client client = new Client(urls);
+
+    /**
+     * Check that we can page 1 by 1 through the {@link CallSet}s we receive from
+     * {@link org.ga4gh.ctk.transport.protocols.Client.Variants#searchCallSets(SearchCallSetsRequest)}.
+     * <p>
+     * The call to retrieve all {@link CallSet}s may return fewer than all of them, subject to
+     * server-imposed limits.  The 1-by-1 paging must enumerate them all, however.  The set of "all"
+     * must be a subset of those gathered one-by-one.
```

Comment:
Because `Utils.getAllCallSets` works by passing `pageSize` = `null` in the `SearchCallSetsRequest`, and that's not guaranteed to return all `CallSet`s.  (Had I known that when first writing these tests, they'd likely be somewhat different.)  The strongest guarantee we can make is that the result of `Utils.getAllCallSets` will contain at least one `CallSet`, and that its `CallSet`s must be contained in the set we collect 1-by-1, since it's guaranteed to collect all `CallSet`s.

I interpreted your earlier comments on the other paging tests to take exactly this tack; did I misunderstand?